redfish-core/include/utils/hw_isolation.hpp

```diff
@@ -141,9 +141,9 @@ inline void
             else if (strcmp(dbusError->name, "xyz.openbmc_project.Common.Error."
                                              "Unavailable") == 0)
             {
-                messages::propertyValueResourceConflict(
-                    aResp->res, "Enabled", std::to_string(enabledPropVal),
-                    "HardwareIsolation");
+                messages::serviceDisabled(
```

Comment:
@raviteja-b just mapped this to Service Unavailable 
https://github.com/ibm-openbmc/bmcweb/pull/301/files

And if I read 
https://github.com/openbmc/phosphor-dbus-interfaces/blob/master/yaml/xyz/openbmc_project/Common.errors.yaml#L17
That mapping makes sense, do we need a Disabled Error?

Reply:
Can we add a xyz.openbmc_project.Common.Error.Disabled ?

Reply:
IMO, an Unavailable error in the D-Bus level is fine for the backend application for this kind of use case.

But, we need to differentiate only on the Redfish side based on the context so I think we can go with it.

Still, you feel we need an error like 1-to-1 mapping i.e Redfish to D-Bus, I think we need to talk more from the design point of view i.e supporting Redfish message registry in the D-Bus or similar design since we can add arguments for the message registry but, (AFAIK) we can not do that in the current D-Bus error design (YAML file).

Reply:
@gtmills  In dump use case, I have mapped  to serviceTemporarilyUnavailable wherever bmcweb has retry time interval for client to retry.
As Ramesh explained seems ServiceDisabled makes more sense as service disabled and there no retry time interval associated in this use-case.
ServiceDisabled error also underlying maps  to boost::beast::http::status::service_unavailable status code.

Reply:
Hi @gtmills / @raviteja-b 

I have removed the code to handle the `Unavailable` error, we don't have support to enable the hardware isolation service through the Redfish for the user. Refer [here](https://github.com/ibm-openbmc/bmcweb/pull/309/commits/6d57fd3a3a494b92e14811bb616bbd0562914a89) to get more details.